From cf334a7d66d46772d7cfd60a95e777f2127509cd Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Sun, 25 Feb 2018 17:42:30 +0800
Subject: [PATCH] arm64: rockchip_cros_defconfig: enable usbmon

Enable usbmon driver to collect traces of I/O on the USB bus.

Change-Id: I7314a2fb01f9ce852e4172aad62ef13fabbd3fd9
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 arch/arm64/configs/rockchip_cros_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_cros_defconfig b/arch/arm64/configs/rockchip_cros_defconfig
index 032116a5a972..45dfc4650197 100644
--- a/arch/arm64/configs/rockchip_cros_defconfig
+++ b/arch/arm64/configs/rockchip_cros_defconfig
@@ -468,6 +468,7 @@ CONFIG_HID_ZEROPLUS=y
 CONFIG_HID_ZYDACRON=y
 CONFIG_USB_HIDDEV=y
 CONFIG_USB_ANNOUNCE_NEW_DEVICES=y
+CONFIG_USB_MON=y
 CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_EHCI_HCD_PLATFORM=y
-- 
2.35.3

